components/Card.tsx

```diff
@@ -3,35 +3,44 @@ import Image from "next/image";
 
 const Card = ({ item }) => {
   return (
-    <div className="flex justify-center mt-5 container mx-auto">
-      <div className="rounded-lg shadow-xl bg-white w-[100%] sm:max-w-[100%] lg:max-w-[300px] ">
+    <div className="flex rounded-xl border bg-gray-50 justify-center w-full mt-5 flex-col lg:flex-row  lg:max-w-[450px] xl:max-w-[420px]">
+      <div className="rounded-lg shadow-xl bg-white w-[100%]  ">
         <a href="#!">
           <Image
-            src="/../public/assects/9.png"
+            // src="/../public/assects/9.png"
+            src={`/QmQqzMTavQgT4f4T5v6PWBp7XNKtoPmC9jvn12WPT3gkSE`}
```

Comment:
change this to https://ipfs.io/ipfs/QmQqzMTavQgT4f4T5v6PWBp7XNKtoPmC9jvn12WPT3gkSE